test/fixtures/bpmn/overlays/overlays.start.flow.task.bpmn

```diff
@@ -0,0 +1,28 @@
+<bpmn:definitions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:bpmn="http://www.omg.org/spec/BPMN/20100524/MODEL" xmlns:bpmndi="http://www.omg.org/spec/BPMN/20100524/DI" xmlns:dc="http://www.omg.org/spec/DD/20100524/DC" xmlns:di="http://www.omg.org/spec/DD/20100524/DI" id="Definitions_0x0opj6" targetNamespace="http://example.bpmn.com/schema/bpmn">
```

Comment:
⚠️ we should remove bpmn element labels as suggested in the testing documentation.
We don't want to test labels here, only overlays. This also impact the threshold we have to configure by os/browsers